chatterbot_corpus/corpus.py

```diff
@@ -88,9 +88,10 @@ def load_corpus(self, dotted_path):
         for file_path in data_file_paths:
             corpus = self.read_corpus(file_path)
 
-            corpora.categories = corpus.get('categories', [])
-            conversations = corpus.get('conversations', [])
+            categories = corpus.get('categories', [])
+            corpora.categories.extend([categories])
```

Comment:
yes, if i trained bot with multiple corpus files,  each it overwrites categories variable at the end i am havong last file category. Fo example

```python
bot.train(
    corpus.data.english
)
```
In this i am getting all english corpus in this scenario the categories are overwrites every time.